spec/javascript/containers/StateIndexConatainerTest.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import StatesIndexContainer from '../../../app/javascript/react/container/StatesIndexContainer'
+import StateTile from '../../../app/javascript/react/components/StateTile'
+import fetchMock from 'fetch-mock'
+import testHelper from '../testHelper'
+import {BrowserRouter} from "react-router-dom"
+
+
+describe('StatesIndexContainer', () => {
+  let wrapper
+  let states
+
+  beforeEach(() => {
+
+    jasmineEnzyme();
+
+    states = [
+      {id: 1, name: 'Alaska', abbrevation: 'AK', description: "This is Alaska!"}
+    ]
+
+    fetchMock.get('/api/v1/states', {
+      status: 200,
+      body: states
+    })
+
+    wrapper = mount(
+      <BrowserRouter>
+        <StatesIndexContainer />
+      </BrowserRouter>
+      );
+  });
+
+  afterEach(fetchMock.restore)
+
+  it('should render a StateTile component', (done) => {
```

Comment:
We need a few more tests here -- see Nick's gist, https://gist.github.com/nwalberts/8e052ef5363da375d92fedf7fd129dd8

(Note that because you've updated jasmine-enzyme, you likely need to use `toExist` instead of `toBePresent`)